### PR TITLE
Added one step install

### PR DESCRIPTION
**Pull request by samghelms**
I updated the setup.py so that you can `pip install cython numpy benepar` all in one command.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -48,4 +51,4 @@
         "gpu": ["tensorflow-gpu>=1.11.0"],
         "spacy": ["spacy>=2.0.9"],
     },
-)
+)
```
Comment (interrogator) [inline setup.py:54]: Newline at end of file would be nicer @samghelms

**Comment by interrogator** [comment]
This seems like a really useful one, as currently it takes my users additional install steps just because of the order of install issue. @nikitakit can you merge? Perhaps just add the newline at end of file here in the browser too?

**Comment by samghelms** [comment]
Done @interrogator

**Comment by nikitakit** [comment]
Thank you for taking the time to look at this installation issue, and sorry that I haven't been monitoring this GitHub repo in a long long time.

In 2021, the wonderful pytorch-struct library has a nice implementation of CKY, and I've updated benepar to not use Cython at all. This has the added benefit of being much faster when using a GPU. Sadly some installation dependency issues remain because pytorch-struct is not on PyPI, but maybe I'll be able to work through those in the future.

**Pull request closed without merging**